### PR TITLE
Updating the events api response for the changed contract

### PR DESCRIPTION
**Pull request by farahInTheZone**

### CODE REVIEW

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json [hunk 1]
```diff
@@ -572,8 +578,11 @@
               "type": "string",
               "description": "It provides the Timestamp for when the health impacting event was last updated.",
               "format": "date-time"
+            },
+            "hIRStage": {
```
Comment (nschonni) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json]: hIR -> hir? not sue if the whole acronym is supposed to be lower case when it leads

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json [hunk 2]
```diff
@@ -572,515 +567,531 @@
...
+                  "Unplanned",
+                  "Planned",
+                  "UserInitiated"
+                ],
+                "x-ms-enum": {
+                  "name": "ReasonTypeValues",
+                  "modelAsString": true
+                }
+              },
+              "occuredTime": {
```
Comment (nschonni) [inline specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json]: It exists already, but if you're changing the contract, "occured" should be "occurred"

**Review by sarangan12 — CHANGES_REQUESTED** [review]
Please fix the errors reported at https://travis-ci.org/Azure/azure-rest-api-specs/jobs/496727704 & https://travis-ci.org/Azure/azure-rest-api-specs/jobs/496727706 for approval

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json (hunk 1 above)
Comment (farahtk) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json]: Is there a way I can confirm what it should be?

File: specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json (hunk 2 above)
Comment (farahtk) [reply specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json]: This path is used by a partner. The areas I am changing for the contract are not in use by any one.

**Comment by farahtk** [comment]
> Please fix the errors reported at https://travis-ci.org/Azure/azure-rest-api-specs/jobs/496727704 & https://travis-ci.org/Azure/azure-rest-api-specs/jobs/496727706 for approval

Fixed.

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**